lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -22,6 +22,13 @@ def self.find(scope, *args)
       end
     end
 
+    def self.release_hold(order_id:)
+      ShopifyAPI::Base.version_validation!('unstable')
```

Comment:
you have to be on unstable for now but before merging this (have to wait for #796 first) I'll update to point to the exact version that will include the change.

The plan is to cut this into RC early next year so it will be in the `2021-04` version of the API.

Reply:
I think you've linked to the wrong PR above.

Reply:
true, I've updated 👍